### ISSUE TITLE
[BUG] : Add aria-labels in Searchbar.tsx File 

### ISSUE THREAD

**Issue by NarendraKoya999** [rupali-codes/LinksHub#1386]
### Description

- **Add `aria -labels` to Form Section in `Searchbar.tsx` File.**
- **Because of The `aria-label `attribute can be added to provide a descriptive label for assistive technologies.** 

### Any additional information?

- ARIA is a W3C specification that stands for “Accessible Rich Internet Applications.” It consists of markup that can be added to HTML in order to communicate the roles, states, and properties of user interface elements to assistive technologies (AT).

### What browser are you seeing the problem on?

Chrome

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

**Comment by Anmol-Baranwal** [comment]
@NarendraKoya999 
Are you still interested in working on this issue? If not, please let me know so that I can open it for others to contribute.

**Comment by NarendraKoya999** [comment]
> @NarendraKoya999 
> Are you still interested in working on this issue? If not, please let me know so that I can open it for others to contribute.

@Anmol-Baranwal I am not interested to work on this Issue. Please do action on that  whatever you want?

**Comment by CBID2** [comment]
@EmmaDawsonDev, you want to take on this issue?

**Comment by EmmaDawsonDev** [comment]
Sure, as long as there's no rush. I should be able to get to it next weekend

**Comment by Anmol-Baranwal** [comment]
I have assigned you the issue. Please work on it whenever you are free. No rush :)
